questions/templates/questions/answers.html

```diff
@@ -68,30 +69,34 @@ <h4 class="my-3">“{{ question.text }}”</h4>
 
     {% if not answers %}
 
-    <div class="col-sm-12">
+    <div class="col-sm-12 col-lg-8">
+
       <h3 class="gray-600" style="text-align: center;">This question hasn’t been answered by any candidates{% if  current_region_readable != "All regions" %} from {{ current_region_readable }} {% endif %} yet. </h3>
     </div>
     
     {% else %}
 
-    <div class="col-sm-12">
+    <div class="col-sm-12 col-lg-8">
       <h2>{{ answers|length }} Answer{% if answers|length > 1 %}s{% endif %}</h2>
     </div>
     
     {% endif %}
 
   </div>
 
+  
+  {% for answer in answers %}
+  
   <div class="row justify-content-center">
+    <div class="col-sm-12 col-md-8">
 
     {% for answer in answers %}
```

Comment:
You now have a nested loop of answers? Probably not going to work, especially since only one of them has an endfor

Reply:
fixed